src/test/java/cow/farm/repo/CowBuilder.java

```diff
@@ -0,0 +1,38 @@
+package cow.farm.repo;
+
+public class CowBuilder {
+	String name = "krasula";
```

Comment:
takie rzeczy zamień na public static final i daj nazwy underscore. Będą to stałe, które łatwo będzie można użyć w asercjach